packages/flutter_tools/lib/src/android/android_device_discovery.dart

```diff
@@ -11,22 +11,29 @@ import '../base/io.dart';
 import '../base/logger.dart';
 import '../base/platform.dart';
 import '../base/process.dart';
+import '../base/user_messages.dart';
 import '../device.dart';
-import '../globals.dart' as globals;
 import 'adb.dart';
 import 'android_device.dart';
 import 'android_sdk.dart';
 import 'android_workflow.dart' hide androidWorkflow;
 
 /// Device discovery for Android physical devices and emulators.
+///
+/// This class primarily delegates to the `adb` command line tool provided by
+/// the Android SDK to discover instances of connected android devices.
+///
+/// See also:
+///   * [AndroidDevice], the type of discovered device.
 class AndroidDevices extends PollingDeviceDiscovery {
   AndroidDevices({
     @required AndroidWorkflow androidWorkflow,
     @required ProcessManager processManager,
     @required Logger logger,
     @required AndroidSdk androidSdk,
-    FileSystem fileSystem, // TODO(jonahwilliams): remove after rolling into google3
-    Platform platform,
+    @required FileSystem fileSystem,
```

Comment:
will clean up with g3fix

Reply:
does that mean this PR is blocked?

Reply:
No, g3 fix allows me to create a CL that has all the needed changes and tell frob about it. Then when this rolls into g3 that CL gets patched into automatically